components/automate-cli/cmd/chef-automate/verify.go

```diff
@@ -204,126 +185,244 @@ func (v *verifySystemdCreateFlow) runVerifySystemdCreateCmd(cmd *cobra.Command,
+		userName = v.Config.Architecture.ExistingInfra.SSHUser
+
+		hostIPs = stringutils.ConcatSlice(hostIPs, v.Config.ExistingInfra.Config.AutomatePrivateIps)
+		hostIPs = stringutils.ConcatSlice(hostIPs, v.Config.ExistingInfra.Config.ChefServerPrivateIps)
+		hostIPs = stringutils.ConcatSlice(hostIPs, v.Config.ExistingInfra.Config.PostgresqlPrivateIps)
+		hostIPs = stringutils.ConcatSlice(hostIPs, v.Config.ExistingInfra.Config.OpensearchPrivateIps)
+
+		sshConfig := sshutils.NewSshConfig("", port, keyFile, userName)
+
+		destFileName := "chef-automate"
+
+		currentBinaryPath, err := v.CreateSystemdService.SystemdCreateUtils.GetBinaryPath()
+		if err != nil {
+			return err
+		}
+
+		v.Writer.Println("Copying automate-verify CLI to remote nodes")
+		copyResults := v.SSHUtil.CopyFileToRemoteConcurrently(sshConfig, currentBinaryPath, destFileName, false, hostIPs)
+		isError := false
+		for _, result := range copyResults {
+			if result.Error != nil {
+				v.Writer.Errorf("Remote copying automate-verify CLI failed on node : %s with error: %v\n", result.HostIP, result.Error)
+				isError = true
 			}
-			return nil
 		}
-		return status.New(status.InvalidCommandArgsError, errProvisonInfra)
+		if isError {
+			return fmt.Errorf("remote copying failed")
+		}
+		v.Writer.Println("Copying automate-verify CLI to remote nodes completed")
+
+		v.Writer.Println("Creating automate-verify service on remote nodes")
+		cmd := fmt.Sprintf("sudo /tmp/%s verify systemd-service create", destFileName)
+		excuteResults := v.SSHUtil.ExecuteConcurrently(sshConfig, cmd, hostIPs)
+		isError = false
+		for _, result := range excuteResults {
+			if result.Error != nil {
+				v.Writer.Errorf("Remote execution of cmd '%s' automate-verify CLI failed on node : %s with error: %v\n", cmd, result.HostIP, result.Error)
+				isError = true
+			}
+		}
+		if isError {
+			return fmt.Errorf("remote execution failed")
+		}
+		// Wait for 2 seconds for the service to start
+		time.Sleep(2 * time.Second)
+		v.Writer.Println("Creating automate-verify service on remote nodes completed")
 	}
-	return status.New(status.InvalidCommandArgsError, "Managed Services flag is not set. Cannot verify the config.")
-}
 
-func (v *verifyCmdFlow) verifyStandaloneDeploy(configPath string) error {
-	if v.A2HARBFileExist {
-		return status.New(status.InvalidCommandArgsError, "Deployment type does not match with the requested flag.")
+	//TODO: Need to handle the case for AWS
+
+	// Doing batch-check API call for remote nodes
+	batchCheckRemoteReq := models.BatchCheckRequest{
+		Checks: constants.GetRemoteChecks(),
+		Config: batchCheckConfig,
 	}
-	if err := v.Verification.VerifyStandaloneDeployment(configPath); err != nil {
-		return status.Annotate(err, status.ConfigError)
+
+	v.Writer.Println("Doing batch-check API call for remote nodes")
+	batchCheckRemoteRes, err := v.Client.MakeRequest("POST", fmt.Sprintf("http://localhost:%s/api/v1/checks/batch-checks", getPort()), batchCheckRemoteReq)
```

Comment:
get endpoint from const, as it is defined in 2 places

Reply:
done